src/components/myOrders/MyOrders.tsx

```diff
@@ -91,66 +51,36 @@ const MyOrders = ({route, navigation}: myOrdersProps) => {
     return orderId;
   };
 
-  const getFormatDate = (date: string) => {
-    Moment.locale('en');
-    return Moment(date).format('DD/MM/YY hh:mm');
-  };
-
-  type orderType = {
-    name: string;
-    title: string;
-    date: string;
-    imagePath: string;
-    total: string;
-    currency: string;
-  };
-
-  const Item = ({name, title, date, imagePath, total, currency}: orderType) => (
-    <View style={styles.priceItemsElements}>
-      {orderIncluded && (
-        <>
-          <Text style={styles.oderDateItem}>{getFormatDate(date)}</Text>
-          <View style={styles.oderItem}>
-            {/*<Text style={styles.oderItemTitle}>{name}</Text>*/}
-            <Text style={styles.oderItemTitle}>{title}</Text>
-            <Text style={styles.oderItemTitle}>
-              {total} {currency}
-            </Text>
-          </View>
-          <View style={styles.orderItem}>
-            <Image
-              style={styles.orderItemImage}
-              source={{uri: `https://demo.spreecommerce.org${imagePath}`}}
-            />
-          </View>
-        </>
-      )}
-    </View>
-  );
-
   return (
     <View style={styles.centeredView}>
       <Bar
         text="My Orders"
         isSearch={true}
         isLike={false}
-        style={shadowStyles}
         isCard={true}
         navigation={navigation}
       />
       <View style={styles.centeredViewScroll}>
-        {orders && (
-          <View style={[styles.orderCard, cardStyles]}>
+        {orders ? (
```

Comment:
As you, possibly, already guessed, such complex parts are better to be written as a separate components, since they do have their own behaviour. It improves component's structure and makes your code more readable.